src/netlink_pm_upstream.c

```diff
@@ -367,17 +367,27 @@ static int upstream_add_addr(struct mptcpd_pm *pm,
           Payload (nested):
               Local address family
               Local address
+              Local port (optional)
               Local address ID (optional)
               Flags (optional)
               Network inteface index (optional)
          */
 
         // Types chosen to match MPTCP genl API.
         uint16_t const family = mptcpd_get_addr_family(addr);
+        uint16_t const port   = get_port_number(addr);
```

Comment:
The definition for `get_port_number()` is missing.  It currently exists as a `static` function in `src/netlink_pm_mptcp_org.c`.

Reply:
Can confirm :)

Reply:
Fixed by commit d983abc.